docs/v0.14/filter_record_transformer.txt

```diff
@@ -154,6 +156,15 @@ A comma-delimited list of keys to keep. Only relevant if `renew_record` is set t
 
 A comma-delimited list of keys to delete.
 
+## FAQ
+
+### What are the differences between `${record["key"]}` and `${key}`?
+
+`${key}` is short-cut for `${record["key"]}`. This is error prone because `${tag}` is unclear for event tag or `record["tag"]`.
```

Comment:
This section is for v0.12. v0.14 has already removed this sytanx, so should change the sentence.